lib/src/main/java/graphql/nadel/validation/NadelFieldValidation.kt

```diff
@@ -40,6 +41,10 @@ class NadelFieldValidation internal constructor(
             .asSequence()
             .let { fieldSequence ->
                 // Apply filter if necessary
+                val areAllFieldsHidden = fieldSequence.all { it.hasAppliedDirective(NadelDirectives.hiddenDirectiveDefinition.name)}
```

Comment:
Can we move this into a method and move the call site up? e.g.

```kotlin
    context(NadelValidationContext)
    fun validate(
        parent: NadelServiceSchemaElement.FieldsContainer,
        overallFields: List<GraphQLFieldDefinition>,
    ): NadelSchemaValidationResult {
        if (areAllFieldsHidden(overallFields)) {
            return …
        }
```

Reply:
Sure, done!